interpreter/src/main/scala/sigmastate/crypto/Platform.scala

```diff
@@ -46,6 +51,12 @@ object Platform {
   /** Returns byte representation of the given field element. */
   def encodeFieldElem(p: ECFieldElem): Array[Byte] = p.value.getEncoded
 
+  /** Returns byte representation of the given point.
```

Comment:
"Returns" is already in the code.  "byte representation of the given point."  is enough

Reply:
Fixed, no problem, but does it really worth to spend efforts on this? I don't think we should be so strict in our reviews. Do you want me to follow the same strict standard, I assume yes?

Reply:
What standard are you following during review? Any link ?

Reply:
This is figuratively speaking "standard". You establish the review rules here, and I try to follow them while reviewing Ergo PRs.

Reply:
And vice versa. Why not to standardize them then? It would be good for sure to avoid comments about "black names" etc